### PR TITLE
DEVOPS-248 Add bezeq prod env  workflow promotion

### PR DESCRIPTION
**Pull request by amar-veego**

### CODE REVIEW

**Review by corest — COMMENTED** [review]
This looks good. But it can break with e.g. veego client and prod env. We don't have all envs for each client. So let's stick then to just `from` and `to` inputs, but make them having full env name like `veego-dev`, or `bezeq-prod` here https://github.com/veegoteam/gitops/pull/91

File: .github/workflows/promote-release.yml [hunk 1]
```diff
@@ -68,12 +72,13 @@ jobs:
         env:
           ENVIRONMENT: ${{ inputs.from }}
           APP: ${{ inputs.app }}
+          CLIENT: ${{ input.client}}
         run: |
           if [ ${ENVIRONMENT} = "dev" ]; then
```
Comment (corest) [inline .github/workflows/promote-release.yml:77]:
```suggestion
          if [ ${ENVIRONMENT} = "veego-dev" ]; then
```

File: .github/workflows/promote-release.yml [hunk 2]
```diff
@@ -68,12 +72,13 @@ jobs:
...
             tag=$(cat charts/${APP}/values.yaml | yq eval-all -j | jq -r .image.tag )
             echo "app-version=${tag}" >> $GITHUB_OUTPUT
           else
-            release="gitops/apps/veego-${{ inputs.from }}/veego/${APP}/release.yaml"
+            release="gitops/apps/${CLIENT}-${{ inputs.from }}/veego/${APP}/release.yaml"
```
Comment (corest) [inline .github/workflows/promote-release.yml:81]:
```suggestion
            release="gitops/apps/${{ inputs.from }}/veego/${APP}/release.yaml"
```

File: .github/workflows/promote-release.yml (hunk 1 above)
Comment (corest) [inline .github/workflows/promote-release.yml:77]:
```suggestion
          if [ ${ENVIRONMENT} = "veego-dev" ]; then
```

**Comment by amar-veego** [comment]
@corest 
can you please confirm that this would be the transition workflow promotion ?
dev -> qa  -> stage -> prod

**Comment by amar-veego** [comment]
@corest

As you said that we don't have many clients. So I have let old setup as it is to avoid confusion, just added simple condition for bezeq-prod.

Please check and confirm.

**Comment by amar-veego** [comment]
@corest I have made changes as you suggested. Please review and confirm.

**Review by corest — APPROVED** [review]

**Pull request merged**